examples/demo/package.json

```diff
@@ -5,13 +5,15 @@
   "scripts": {
     "android": "react-native run-android",
     "ios": "react-native run-ios",
+    "macos": "react-native run-macos",
     "lint": "eslint .",
     "start": "react-native start",
     "test": "jest"
   },
   "dependencies": {
     "react": "18.2.0",
     "react-native": "0.72.4",
+    "react-native-macos": "~0.73.30",
```

Comment:
This is the latest version of `react-native-macos`.

Although `react-native-macos` does provide iOS and Android implementations as well as macOS, I've avoided swapping `react-native` for `react-native-macos`, as I see there is ongoing work on New Architecture support and I don't want `react-native-macos` to block that with its lagging update cycle.

Reply:
I would keep the version of RN and RNM the same minor, so that they don't bring in conflicting transitive dependencies like metro, the CLI, etc..